pyi.py

```diff
@@ -1150,45 +1148,23 @@ def visit_Call(self, node: ast.Call) -> None:
         for kw in node.keywords:
             self.visit(kw)
 
-    def _check_for_Y053(self, node: ast.Constant | ast.Str | ast.Bytes) -> None:
-        if len(node.s) > 50:
-            self.error(node, Y053)
-
-    def _check_for_Y054(self, node: ast.Constant | ast.Num) -> None:
-        # The maximum character limit is arbitrary, but here's what it's based on:
-        # Hex representation of 32-bit integers tend to be 10 chars.
-        # So is the decimal representation
-        # of the maximum positive signed 32-bit integer.
-        # 0xFFFFFFFF --> 4294967295
-        if len(str(node.n)) > 10:
-            self.error(node, Y054)
-
-    # 3.8+
     def visit_Constant(self, node: ast.Constant) -> None:
         if isinstance(node.value, str) and not self.string_literals_allowed.active:
             self.error(node, Y020)
         elif isinstance(node.value, (str, bytes)):
-            self._check_for_Y053(node)
+            if len(str(node.value)) > 50:
```

Comment:
That seems wrong for bytes, or at least it's a change in behavior.

Reply:
No, our current behaviour is to treat `str` and `bytes` the same when it comes to Y053:

https://github.com/PyCQA/flake8-pyi/blob/e8582f43e79fa4f284a1c454034f4450048b67e2/pyi.py#L1166-L1171

https://github.com/PyCQA/flake8-pyi/blob/e8582f43e79fa4f284a1c454034f4450048b67e2/pyi.py#L1182-L1184

We assert as such in our tests:

https://github.com/PyCQA/flake8-pyi/blob/e8582f43e79fa4f284a1c454034f4450048b67e2/tests/defaults.pyi#L73

https://github.com/PyCQA/flake8-pyi/blob/e8582f43e79fa4f284a1c454034f4450048b67e2/tests/defaults.pyi#L76

Reply:
We're currently using `len(bytes)`, not `len(str(bytes))`. They're different.

Reply:
> We're currently using `len(bytes)`, not `len(str(bytes))`. They're different.

...good point.

Reply:
```suggestion
            if len(node.value) > 50:
```